README.md

```diff
@@ -10,7 +10,7 @@ Is this a complete solution? No. This fork has a lot of disadvantages like requi
 
 Thankfully `udev` has stable API and hopefully no changes will be made to it the future. On this basis I decided to create this clean-room implementation of `libudev` which can be used with any or without a device manager.
 
-[1] https://github.com/FreeBSDDesktop/libudev-devd  
+[1] https://github.com/FreeBSDDesktop/libudev-devd
```

Comment:
My editor is setup up to remove all trailing space so this change was just something I didn't see the need to revert. Am I understanding correctly that you want left the space here intentionally? Or do you just not want this commit to change it since the space is harmless and irrelevant.

Reply:
Just do `git restore README.md`. Spaces are required for newline

Reply:
I already reverted the space changes although I don't believe spaces matters for links like this and when they do actual newlines may be less subtle.